lib/stats/series.js

```diff
@@ -0,0 +1,209 @@
+import base from 'taskcluster-base';
+
+const BASE_WORKER_SCHEMA = {
+    workerId: base.stats.types.String,
+    workerType: base.stats.types.String,
+    workerGroup: base.stats.types.String,
+    provisionerId: base.stats.types.String
+};
```

Comment:
Ok, I can definitely add that.  That was one of the reasons I abstracted this out so it was easy to change and report it for all metrics.